web/templates/popovers/left_sidebar/left_sidebar_views_popover.hbs

```diff
@@ -3,5 +3,25 @@
         {{#each views}}
             {{> left_sidebar_view_popover_item .}}
         {{/each}}
+        <li role="separator" class="popover-menu-separator"></li>
+        {{#if unread_messages_present}}
+        <li role="none" class="link-item popover-menu-list-item">
+            <a role="menuitem" id="mark_all_messages_as_read" class="popover-menu-link" tabindex="0">
```

Comment:
I note that we use this `#mark_all_messages_as_read` ID across a number of left sidebar popovers. While that's probably okay, given that we only ever show a single popover at a time, I wonder if it wouldn't be wise to take the time to write a prep commit that changes that ID to a class, since IDs are not meant to be used more than once.